.github/actions/rerun/action.yml

```diff
@@ -0,0 +1,42 @@
+# The Rerun Command action is a custom GitHub Actions composite module designed to
+# execute a shell command with optional environment variables and retry logic.
+#
+# This is particularly useful for scenarios such as flakey tests or intermittent runner issues,
```

Comment:
I'm not keen on adding this to EXIPS, we rarely have flakey tests.

I feel that adding this would only introduce additional time to get PR test execution results. I.e if some E2E tests fail, they will run again until it's reported that they fail so it'll be double the time to get results.

99% of the time in EXIPS, if tests fail it is a genuine issue. The 1% is either external APIM rate limiter issues, if there are a lot of PRs running tests.

I cannot see a value in introducing re-runs?

Reply:
This is already part of EXIP, I have just renamed the file and updated the script.

This will only re-run failed tests.